tests/components/modbus/test_modbus.py

```diff
@@ -21,6 +22,19 @@
 from .conftest import base_config_test
 
 
+def test_number_helper():
+    """Test number method."""
+    assert number_helper(5) == 5
+    assert number_helper(5.1) == 5.1
+    assert number_helper("5") == 5
+    assert number_helper("5.1") == 5.1
+
```

Comment:
This does not seem to have anything to do with bit_sensor/bit_switch.

Reply:
just a codecov complain

Reply:
That sounds a bit unlikely. Codecoverage does not complain in branch dev, apart from the fact that this test is already active in test_init.py

Reply:
Removed